src/sass/landingPage.scss

```
@@ -200,6 +201,107 @@ $img-base-url: 'https://d1yw74buhe0ts0.cloudfront.net/static/motrpac-data-hub/im
+    position: relative;
+
+    @media (min-width: 640px) {
+      text-align: left;
+    }
+  }
+
+  .tribute-eyebrow {
+    display: block;
+    color: rgba(100, 85, 60, 0.85);
+    @include fluid-text(0.75rem, 1.2vw, 0.9rem);
+    font-weight: 600;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)